modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -423,9 +423,6 @@ protected function register_controls() {
 				'exclude' => [ 'image' ],
 				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title",
 				'fields_options' => [
-					'background' => [
-						'default' => 'classic',
-					],
```

Comment:
Why is this removed?

Reply:
I added this by mistake, AFAIR it doesn't appear in the original branch